freezeyt/freezer.py

```diff
@@ -239,12 +239,28 @@ class Freezer:
     saver: Saver
 
     def __init__(self, app, config):
-        self.app = app
         self.config = config
         self.check_version(self.config.get('version'))
 
         self.freeze_info = hooks.FreezeInfo(self)
 
+        module_name = config.get('module_name')
+        if app is None:
+            if module_name is None:
+                raise ValueError("Application is required")
+
+            if isinstance(module_name, str):
+                self.app = import_variable_from_module(
+                    module_name, default_variable_name='app',
+                )
+            else:
+                # from config variable
+                self.app = module_name.app
```

Comment:
We can import app with different name (eg wsgi_application) in config file, but we cannot import app with different name by config variable. At least our README is lacking in this case.